scripts/update-contributors.pl

```diff
@@ -26,6 +27,16 @@
 my $SkippedEmptyLines = 0;
 my $SkippedBadLines = 0;
 
+# Brief display by default.
+# Use --quiet for no output
+# Repeat -v or --verbose for more details
+my $VerboseOutput = 1;
+
+GetOptions(
+    'quiet' => sub { $VerboseOutput = 0 },
+    'verbose+' => \$VerboseOutput, 'v+' => \$VerboseOutput,
+    );
```

Comment:
Dying on errors is the only change I insist on:

```suggestion
    ) or die("$0: Bad command line arguments\n");
```

I do not insist on any specific error message spelling.

N.B. GetOptions() reports individual errors so we do not need to detail which arguments are bad:

```
Unknown option: verbosed
```